src/nvim/runtime.c

```diff
@@ -970,21 +965,22 @@ static int add_pack_dir_to_rtp(char *fname, bool is_pack)
   // Find "ffname" in "p_rtp", ignoring '/' vs '\' differences
   // Also stop at the first "after" directory
   size_t fname_len = strlen(ffname);
-  char *buf = try_malloc(MAXPATHL);
-  if (buf == NULL) {
-    goto theend;
-  }
+  char buf[MAXPATHL];
   const char *insp = NULL;
   const char *after_insp = NULL;
-  for (const char *entry = p_rtp; *entry != NUL;) {
+  const char *entry = p_rtp;
+  while (*entry != NUL) {
     const char *cur_entry = entry;
-
     copy_option_part((char **)&entry, buf, MAXPATHL, ",");
 
-    if ((p = strstr(buf, "after")) != NULL
-        && p > buf
-        && vim_ispathsep(p[-1])
-        && (vim_ispathsep(p[5]) || p[5] == NUL || p[5] == ',')) {
+    char *p = strstr(buf, "after");
+
```

Comment:
What is the point of this blank line?
```suggestion
```

Reply:
The same as most blank lines? Mainly making the code less cramped. I can remove this specific one if it's a problem.

Reply:
> Mainly making the code less cramped

That's a weak argument. Adding a blank line weakens the indication that two statements are related, but the statements above and below this are carefully closed related.